internal/slack/files_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package slack
+
+import (
+	"archive/zip"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO make it so that this doesn't actually reach out to Slack's servers..
```

Comment:
Thanks for the comment on this as I could see this maybe biting us in the future.